Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -2223,11 +2223,12 @@ public static int GetSpellEnchantPtCost(int spellIndex)
         }
 
         /// <summary>
-        /// Reversed from classic. Calculates cost of casting a spell.
-        /// For now this is only being used for enchanted items, because there is other code for entity-cast spells.
```

Comment:
Please leave this comment in place. Spell point costs really do have a different formula for entity-cast spells that emulates the classic formula while allowing modded effects to set their costs. In DFU it's possible to have any number of custom spells effects, and they need to provide costings too.

At some point enchantment costings need to be moved over the new formula anyway, as it's currently limited to the classic effect only. This is another TODO item I need to loop back to as part of spell mod support.

Reply:
No problem, I restored the comment. While doing so, I also removed the last element of magic effect `settingsTypes` array in `CalculateCastingCost` since it is supposed to be an array of 51 bytes, not 52. This matches classic and was probably a bad copy/paste from the original exe data.

Reply:
Thank you. :)